paper.tex

```diff
@@ -516,19 +520,31 @@ \subsubsection{Conceptualization}
 One promising possibility is to start with two positions. The first one would be an RSE coordinator.
-They will be the contact person for all institutional RSEs, organizing meetings, developing training programs, reporting to superordinate bodies, etc.
+They will be the contact person for all institutional RSEs (whether belonging to the hub department or to individual research groups, i.e., decentralised network of RSEs in the institution), organising meetings, developing training programs, reporting to superordinate bodies, etc.
 The second position would be a central RSE, responsible for providing selected services and infrastructure.
-These central positions will be complemented by the existing RSEs organized in the network to form a pool of institutional RSEs associated with the department.
+These central positions will be complemented by the existing RSEs organised in the network to form a pool of institutional RSEs associated with the department.
 
 Drawing from the concept and considering the actual initial staff situation, a first task of the centrally funded structure is to define a basic service portfolio according to the modules described in \autoref{sec:vision}.
 In addition to the already mentioned networking and teaching cf. \autoref{sec:network} and \autoref{sec:teaching}, it seems natural to start with consultation cf. \autoref{sec:consultation}, as this allows to evaluate the potential necessities for other services such as development, infrastructure provisioning and maintenance.
-An extension of the initial service portfolio for a larger target audience requires the acquisition of further funding and installation of further positions, see below.
+An extension of the initial service portfolio for a larger target audience requires the acquisition of further funding and installation of further positions, see below (\autoref{sec:growth_of_dep}).
+
+\subsubsection{Internal communication of the department}
+The best RSE department can easily become useless if other departments as well as RSEs outside the central hub do not know about it. 
+Although an initial announcement will be done once the the RSE department concept has been approved by the institution, communication across the RSE hub, the RSE network and other departments should be done on a regular basis. 
+Such a communication should facilitate collaboration among RSEs while also keeping non-RSEs in the loop. 
+
+For communication among RSEs, some options include messaging channels (for quick communication) and forums or wikis integrated to developer platforms to have longer discussion while keeping track of the conversation, e.g.,  via issues. 
+Project management communication tools, e.g., Kanban boards, could complement the picture. 
+
+With regards to the broader community, i.e., departments within the corresponding institute, a portal including the portfolio of services and training materials around best RSE practices, would be ideal. 
```

Comment:
I am not sure I like the term "portal" here. To me, that is an "interactive web service providing some functionality". I disagree that, besides possibly an own web page, an RSE group should in all cases have a separate "portal". In some cases, maybe even often, it would be better to integrate in already existing and used infrastructure to be visible there. I also tend to be very careful with the word "ideal" personally.
I am not really sure what you wanted to say here: we should quickly discuss this in a call.

Reply:
Updated to

With regards to the broader community, i.e., departments within the corresponding institute, a web/wiki page including the portfolio of services and training materials around best RSE practices, would be ideal.